### DISCUSSION TITLE
Collaboration Workflow

### DISCUSSION THREAD

**Discussion by sbstjn** [superwerker/superwerker#161] [Ideas] [answered]
Based on #160, I would like to propose the following workflow:

- Use GitHub issues mainly for "Bug Reports" using custom template and disable "blank issues" (using template in #160)
- Try to force people to use Discussions for every non-bug issue (using config.yml in #160)
- Provide `SECURITY.md` file with basic information

**General**

- Discuss features, changes, roadmap plans etc. in GitHub Discussions
  - rule of thumb: _Create a discussion for everything that's not clearly a bug._
- Use GitHub Projects with high-level cards to manage process and plans
  - rule of thumb: _Have no Issues as "cards" in projects._

**Attachments**

<img width="983" alt="Screenshot 2021-02-22 at 10 53 37" src="https://user-images.githubusercontent.com/248965/108691894-46014100-74fc-11eb-918c-9d465957252c.png">

**Comment by mavogel** [comment, marked as answer]
@sbstjn very good. I like. Let's go with this approach!

**Reply by sbstjn** [reply]
I'll talk to @yawn tomorrow (about this and other stuff), and merge the PR afterward if he has no strong opinion against it.

**Reply by yawn** [reply]
Sounds like a great solution, let's do it like this.
